crowbar_framework/app/views/nodes/_show.html.haml

```diff
@@ -18,7 +18,7 @@
       = render :partial => "show_barclamps"
       = render :partial => "show_roles"
 
-  - if @node.bmc_set? or not @node.admin?
+  - if (@node.bmc_set? or not @node.admin?) && @node.state != "crowbar-upgrade"
```

Comment:
See https://github.com/crowbar/barclamp-crowbar/pull/1358#issuecomment-166855171

If user turns off the node using the Power button, it's state stays as the power action selected.